src/aat/resources/features/F-040/F-040_Later_Case_Read_By_Solicitor_1.td.json

```diff
@@ -0,0 +1,166 @@
+      "id": "[[ANY]]",
+      "jurisdiction": "BEFTA_JURISDICTION_2",
+      "state": "TODO",
+      "version": 0,
+      "case_type_id": "BEFTA_CASETYPE_2_1",
+      "created_date": "[[ANY]]",
+      "last_modified": "[[ANY]]",
+      "security_classification": "PUBLIC",
+      "case_data": {
+        "DocumentField4": {
+          "document_url": "http://dm-store-aat.service.core-compute-aat.internal/documents/e56f0aea-4a5f-46d5-82d3-01b536420870",
+          "document_filename": "Screenshot 2019-11-14 at 1.57.32 pm.png",
+          "document_binary_url": "http://dm-store-aat.service.core-compute-aat.internal/documents/e56f0aea-4a5f-46d5-82d3-01b536420870/binary"
+        },
+        "MoneyGBPField": "111100",
+        "FixedListField": "VALUE3",
+        "AddressUKField": {
+          "County": null,
+          "Country": null,
+          "PostCode": null,
+          "PostTown": null,
+          "AddressLine1": null,
+          "AddressLine2": null,
+          "AddressLine3": null
+        },
+        "DocumentField5": {
+          "document_url": "http://dm-store-aat.service.core-compute-aat.internal/documents/f96735a6-8764-4778-978c-e7612a610a94",
+          "document_filename": "Screenshot 2019-11-18 at 10.20.24 am.png",
+          "document_binary_url": "http://dm-store-aat.service.core-compute-aat.internal/documents/f96735a6-8764-4778-978c-e7612a610a94/binary"
+        },
+        "DocumentField2": {
+          "document_url": "http://dm-store-aat.service.core-compute-aat.internal/documents/ae5c9e4b-1385-483e-b1b7-607e75dd3943",
+          "document_filename": "Elastic Search test Case.png",
+          "document_binary_url": "http://dm-store-aat.service.core-compute-aat.internal/documents/ae5c9e4b-1385-483e-b1b7-607e75dd3943/binary"
+        },
+        "ComplexField": {
+          "ComplexTextField": null,
+          "ComplexNestedField": {
+            "NestedNumberField": null,
+            "NestedCollectionTextField": []
+          },
+          "ComplexFixedListField": null
+        },
+        "FixedRadioListField": "RADIOVALUE2",
+        "DateTimeField": null,
+        "PhoneUKField": null,
+        "DocumentField7": {
+          "document_url": "http://dm-store-aat.service.core-compute-aat.internal/documents/5c14f8ef-887b-41e0-ad01-23959c082311",
+          "document_filename": "Screenshot 2019-12-24 at 3.41.04 pm.png",
+          "document_binary_url": "http://dm-store-aat.service.core-compute-aat.internal/documents/5c14f8ef-887b-41e0-ad01-23959c082311/binary"
+        },
+        "NumberField": "111111222222",
+        "MultiSelectListField": [
+          "OPTION5",
+          "OPTION4",
+          "OPTION3"
+        ],
+        "YesOrNoField": "Yes",
+        "EmailField": "testemail@gmail.com",
+        "TextField": "sometextfield",
+        "DateField": "2222-01-02",
+        "TextAreaField": "textarea",
+        "CollectionField": []
+      },
+      "data_classification": {
+        "DocumentField4": "PUBLIC",
```

Comment:
Can use inheritance by creating a common file. Please check the feasibility.

Reply:
Siva, it is being used at only one place. So we can skip the inheritance for now.